### PR TITLE
First review

### PR DESCRIPTION
**Pull request by bbakernoaa**
@btang1 I'm just adding a few comments and restructure here. But just look through the differences here and address some of the comments.  We can chat about them if you'd like

### CODE REVIEW

File: module_vd_aero_wesley.f90 [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+      rd = 1./(ustar*(sc23 + xinert))
+      rd = amax1(rd, rmin)
+
+      !Step 5. Calculate aerosol depostion velocity for this cell, land use, and aerosol size
+      vd = vsed + 1./(ra + rd + ra* rd * vsed)
+
+      return
+
+   end subroutine Vd_Aerosol_Wesely
```
Comment (btang1) [inline module_vd_aero_wesley.f90:76]: should we use "end subroutine aero wesely" here?

File: module_vd_gas_wesely.f90 [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+        real, intent(in)    :: psih
+        real, intent(in)    :: ustar
+        real, intent(in)    :: diffrat
+        real, intent(in)    :: henry
+        real, intent(in)    :: henso2
+        real, intent(in)    :: f0
+        real, intent(in)    :: rscale
+        real, intent(in)    :: ts
+        real, intent(in)    :: dstress(0:5)
+        real*8, intent(in)  :: solflux ! why does this one have precision but the others dont?
```
Comment (btang1) [inline module_vd_gas_wesely.f90:76]: I just copy from CAMx code, they have a double precision for solflux. 
Also, in CAMx code, they only define solflux and diffrat here. I added all other input and output parameter types here.

File: module_vd_gas_wesely.f90 [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+        !!Step 3-1. special case surface layer over water
+        if (ilu == 7) then
+            if (io3 == 1 .and. icdocn == 1) then
+                rs = 1./(1.e-4 + 5.e-6 * henry *ustar * ts **3)                                   !Follow Helmig et al (2012)
+                rs = amax1(rs,1500.)
+            else
+                rs = 1./ (3.9e-5*henry*ustar*(ts+273.14))                                         !Follow Kumar et al (1996) and Sehmel (1980)
+                rs = amax1(rs,rmin)
+            endif
+            goto 100 ! need to remove goto blocks - they are not accepted by NCO
```
Comment (btang1) [inline module_vd_gas_wesely.f90:105]: sure! should I use if else structure instead?

File: module_vd_gas_wesely.f90 [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+        if (iwet == 0) then                     !dry surface
+            ruc = rlu/(henry/henso2+f0)
+            ruc = amin1(rmax,ruc)
+        else
+            if (iwet == 1) then                 !dew surface
+                rlus = 100.
+                if (ilu == 1) then
+                    rlus = 50.
+                endif
+    !            rluo = 1./(1./3000.+1./(3.*rlu))                                                   !Follow Wesely et al.,(1989) eqn (11)
```
Comment (btang1) [inline module_vd_gas_wesely.f90:137]: another question is why in CAMx code they turn off wesely formula, and used simple 1000+rlu formula instead here?

File: module_vd_gas_wesely.f90 [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+
+        !!!Step 3-2-3.buyant convection and lower canopy resistance, Rdc & Rlc
+        rdc = 100.*(1. +1000./(solflux +10.))                  !Follow Wesely (1989) eqn (5),but set terrain slope factor to be 1. 
+        rdc = amin1(rmax, rdd)
+
+        rlc = 1./(henry/(henso2*rlcs)+f0/rlco)                 !Follow Wesely et al. (1989) eqn (8)
+        rlc = amin1(rmax,rlc)
+
+        !!!Step 3-2-4.ground surface resistance, Rgs
+        !!!Rac was read-in ???
```
Comment (btang1) [inline module_vd_gas_wesely.f90:166]: rac is read in?

File: module_vd_gas_wesely.f90 [hunk 6]
```diff
@@ -0,0 +1,211 @@
...
+        rgs = 1./(henry/(henso2*rgss) * f0/rgso)               !Follow Wesely et al., (1989) eqn (9)
+        rgs = amin1(rmax,rgs)
+
+        !Step 4. Calculate total resistance
+        rs = 1./ (rst + rm) + 1./ruc + 1./(rdc + rlc) + 1./(rac +rgs) 
+        rs = amax1(rmin, 1./rs)
+
+
+100 continue ! need to remove continue blocks
```
Comment (btang1) [inline module_vd_gas_wesely.f90:175]: sure! I will remove goto and continue structure

File: module_vd_aero_wesley.f90 (hunk 1 above)
Comment (bbakernoaa) [reply module_vd_aero_wesley.f90:76]: yes it should be `end subroutine aero_wesley`

File: module_vd_gas_wesely.f90 (hunk 3 above)
Comment (bbakernoaa) [reply module_vd_gas_wesely.f90:105]: yes

File: module_vd_gas_wesely.f90 (hunk 4 above)
Comment (bbakernoaa) [reply module_vd_gas_wesely.f90:137]: I'm not sure.  maybe rick will know a better answer

File: module_vd_gas_wesely.f90 (hunk 5 above)
Comment (bbakernoaa) [reply module_vd_gas_wesely.f90:166]: see line 42.  You have it listed as an input variable

**Pull request merged**